doc/usermanual/darkroom/modules/correction/denoise_profiled.xml

```diff
@@ -115,11 +137,34 @@
       <title>search radius</title>
       <para>
         This slider is only available if mode <quote>non-local means</quote> is selected. It
-        controls the how far from a pixel the algorithm will try to find similar patches. The
+        controls how far from a pixel the algorithm will try to find similar patches. The
         processing time is hugely impacted by this parameter: it depends on the square of the
         parameter. A lower value will make execution faster, a higher value will make it slower.
         Increasing the value can give better results for very noisy images when coarse grain
-        noise is visible.
+        noise is visible, but you should better use the scattering slider instead.
+      </para>
+    </sect5>
+    
+    <sect5>
+      <title>scattering</title>
+      <para>
+        This slider is only available if mode <quote>non-local means</quote> is selected. Like
+	the search radius, it controls how far from a pixel the algorithm will try to find similar
+	patches, but does this without increasing the number of patches considered. As such,
+	processing time will stay about the same. Increasing the value will reduce coarse grain
+	noise, but may smooth local contrast. This slider is particularly effective to reduce
+	chroma noise.
+      </para>
+    </sect5>
+    
+    <sect5>
+      <title>central pixel weight</title>
+      <para>
+	This slider is only available if mode <quote>non-local means</quote> or <quote>non-local means
+	auto</quote> is selected. It controls the amount of details which should be preserved by the
+	algorithm. It can be used as a way to control the amount of luma noise smoothing: giving a big
+	value to this slider will result mostly in chroma noise smoothing with little smoothing of chroma
+	noise. This slider has no effect if patch-size is set to 0.
```

Comment:
Doest this makes sense: ...chroma noise smoothing with little smoothing of chroma noise...